tests/handlers/test_distgit.py

```diff
@@ -85,3 +85,39 @@ def test_distgit_push():
     assert child is not None
 
     assert child.parent.is_connected(parent)
+
+
+def test_parse_bugzilla_bugs():
```

Comment:
Instead of calling the function multiple times, could you please parametrize this with pytest? The issue with the way it is now is that if the first assert fails, we don't know if the other asserts would have failed. It helps having them separate for debugging purposes.